dune

```diff
@@ -9,3 +9,13 @@
  (name vodeps)
  (deps tools/coq_dune.exe .vfiles.d))
   ; (action (run coq_dune .vfiles.d))))
+
+(install
+ (section lib)
+ (files
+   revision))
+
+(rule
+ (targets revision)
+ (deps (:rev-script dev/tools/make_git_revision.sh))
+ (action (with-stdout-to revision (run %{rev-script}))))
```

Comment:
I didn't find where this was documented but is the script going to be run at every build regardless of whether `dev/tools/make_git_revision.sh` changed or not? Because if that's not the case, then the output of `coqtop | echo` might become outdated, mightn't it?

Reply:
This has roughly the same semantics than the current makefile, I think, which indeed are problematic as they can go out of date as you point out (see #483).

The file will be created once thus. That works for the OPAM build, not so well for the dev build, but this issue is global, I refer to the discussion on the several issues about versioning.

Reply:
OK I was not aware that the make-based system suffered the same issue.